docker-compose.yml

```diff
@@ -21,8 +21,8 @@ services:
 
     environment:
       # Should the web interface use HTTPS or HTTP?
-      # Valid entries are "true" and "false" (without quotes)
-      USE_HTTPS: true
+      # Valid entries are "true" and "false" (with quotes)
+      USE_HTTPS: "true"
```

Comment:
leaving the quotes off causes a syntax error "services.mineos.environment.USE_HTTPS must be a string, number or null"